### DISCUSSION TITLE
Welcome to learning-path Discussions!

### DISCUSSION THREAD

**Discussion by MARCHINGO** [Connect-with-team/learning-path#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MARCHINGO** [comment]
@Parva2005  hello bro give me some best html and CSS tutorial from you tube so we i add that in to the learning path

**Reply by Parva-Shastri** [reply]
We will focus on DOCS only. Videos and tutorials will be prohibited as its only for personal guidance only!

**Reply by MARCHINGO** [reply]
yes bro our first goal is docs only and the tutorial is optional things and when a beginners start learning IT related things so if docs is getting make them boring so there is some chance that they stop learning. So think twice and provide both thinks two them and when they become more handy then we provide just docks ok bro. So give some tutorial if you have any.

**Reply by Parva-Shastri** [reply]
Kale explain karis! Aapde bore thai java wada aapdi team ma joitaj nathi... Tutorials provide nai kariye, koi pan roadmap ma tutorials nai aave. That's it 👍🏻
